geode-core/src/main/java/org/apache/geode/internal/net/NioSslEngine.java

```diff
@@ -396,9 +402,17 @@ public synchronized void close(SocketChannel socketChannel) {
     } catch (IOException e) {
       throw new GemFireIOException("exception closing SSL session", e);
     } finally {
-      bufferPool.releaseBuffer(TRACKED_SENDER, myNetData);
-      bufferPool.releaseBuffer(TRACKED_RECEIVER, peerAppData);
+      logger.info("NioSSLEngine releasing two buffers myNetData={}({}), peerAppData={}({})",
```

Comment:
This should be info or debug level log? just checking...

Reply:
thanks for catching that Anil - that's debug output that I meant to delete